src/polymer/iron-flex-layout-classes.ts

```diff
@@ -214,6 +198,37 @@ Import it in the template style include.`,
   return missingModules.trim();
 }
 
+function searchUsedModulesAndIncludes(
```

Comment:
Using Objects as maps is an anti-pattern. Recommend `Map<string, dom5.Node[]>` instead for `modules`